packages/rpc-graphql/src/__tests__/account-test.ts

```diff
@@ -8,7 +8,7 @@ import {
 import { createHttpTransport, createJsonRpc, type Rpc } from '@solana/rpc-transport';
 import fetchMock from 'jest-fetch-mock-fork';
 
-import { createRpcGraphQL, RpcGraphQL } from '../rpc';
+import { createRpcGraphQL, RpcGraphQL } from '..';
```

Comment:
I think it's _slightly_ preferable to make this explicit, though I don't have a good reason why.
```suggestion
import { createRpcGraphQL, RpcGraphQL } from '../index';
```

Plz disagree.

Reply:
Yeah I have no idea tbh, but explicit looks better.